likecoin/admin/post.php

```diff
@@ -22,6 +22,147 @@
 
 // phpcs:disable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
 
+/**
+ * Require files
+ */
+require_once dirname( __FILE__ ) . '/../public/likecoin-button.php';
+
+/**
+ * Get default style defined in matters library.
+ * Refer to https://github.com/thematters/matters-html-formatter/blob/main/src/makeHtmlBundle/formatHTML/articleTemplate.ts for details.
```

Comment:
This link is dead?

Reply:
Seems they have refactored the lib and I can't easily find the corresponding latest version, lets not mind it for now...